src/util/logger.js

```diff
@@ -44,7 +44,6 @@ function resetOnce(key) {
 export function checkIsDev() {
 	return isDevMode &&
 		typeof window !== 'undefined' &&
-		!isNode &&
```

Comment:
I removed this because of the tests for `logger`. They were failing the `isDevMode` check because of running in node.

I don't think we actually need it here but I don't know why it was added originally.